app/views/admin/facility_groups/_form.html.erb

```diff
@@ -41,7 +40,7 @@
       <div class="mt-3">
         <label><%= I18n.t("helpers.label.facility.zone").tr("* ", "").pluralize %></label>
         <div class="input-group mb-2">
-          <input type="text" class="form-control" id="new-block-name" placeholder="Add new <%= I18n.t("helpers.label.facility.zone").tr("* ", "") %>">
+          <input type="text" class="form-control" id="new-block-name" placeholder="Add new <%= I18n.t("helpers.label.facility.zone").downcase.tr("* ", "") %>">
```

Comment:
If we're not getting the Rails-automagic label from the specific location of this locale `helpers.label.<model>.<attr>`, then shall we use top-level `facility_zone` locale instead? The way I see it now, the top-level locales you've added are the generic keys we should use across the dashboard, and the `helpers.` keys are added _just_ to leverage some Rails magic. 

What do you think? I want to be fairly mindful of how we're using which keys, since we're introducing a little redundancy with this PR.

Reply:
Agree, we should have high level localization and, if needed, make the helpers use *those*. Will clean that up, perhaps in another PR.